packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -108,8 +109,8 @@ export class StripeHelper {
   private redis: ioredis.Redis | undefined;
   private statsd: StatsD;
   private taxIds: { [key: string]: string };
-  private firestore: Firestore;
-  private stripeFirestore: StripeFirestore;
+  private firestore?: Firestore;
+  private stripeFirestore?: StripeFirestore;
```

Comment:
Why are these optional? I guess we do have a `config.authFirestore.enabled` flag; is this temporary until we make sure it all works?

What's the difference between `config.firestore` and `config.authFirestore` in `fxa-auth-server/config/index.ts`?

Reply:
I'm not entirely sure that the local firestore emulator provided by Google is capable of doing collectionGroup queries. I've had a tough time using some features with it, since it also doesn't run its own web interface like the production system. So having this optional allows us to turn it off for local development so that it isn't an additional hassle for every developer.

Reply:
Regarding `config.firestore` vs `config.authFirestore`, we previously had scripts in auth-server that needed to talk to the event-broker firestore config to load user login data up. The scripts read `config.firestore` which is the event-broker firestore config. I prefixed this to designate this as a different firestore config.